CoffeeChat/Utilities/Time+Helper.swift

```diff
@@ -8,15 +8,39 @@
 
 import Foundation
 
+enum Weekday: String {
+    case monday, tuesday, wednesday, thursday, friday, saturday, sunday
+
+    static func fromString(_ day: String) -> Weekday {
+        let day = day.lowercased()
+        if let weekday = Weekday(rawValue: day) {
+            return weekday
+        } else {
+            print("Tried to create a Weekday with \(day); returning .monday instead")
+            return .monday
+        }
```

Comment:
Does this behavior provide any additional benefits compared to being an optional Weekday, or is Monday always the best day to return (compared to say, the 6th day from now)?